plugins/battery/battery.plugin.sh

```diff
@@ -48,6 +55,13 @@ ac_adapter_disconnected(){
   then
     WMIC Path Win32_Battery Get BatteryStatus /Format:List | grep -q 'BatteryStatus=1'
     return $?
+  else
+  	if [[ "$(cat /sys/class/power_supply/ADP0/online)" == "0" ]] ;
+      then
+        true
+ 	    else
+ 		    false
+ 	  fi
```

Comment:
I think this will output error messages in the terminal in the systems without the file. The same applies to the other changes.

Also these six lines are actually equivalent to just
```suggestion
  	[[ "$(cat /sys/class/power_supply/ADP0/online)" == "0" ]]
```